third_party/typings.bzl

```diff
@@ -25,6 +25,10 @@ def tensorboard_typings_workspace():
               "http://mirror.bazel.build/raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/ebc69904eb78f94030d5d517b42db20867f679c0/chai/chai.d.ts",
               "https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/ebc69904eb78f94030d5d517b42db20867f679c0/chai/chai.d.ts",
           ],
+          "a285ca43837c03640134d31fb64a52625f65f4a2890194414d695fbc050b289e": [
+              "http://mirror.bazel.build/raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/5d0f2126c8dac8fce0ff020218aea06607213b0d/google.analytics/ga.d.ts",
```

Comment:
With the "just mock out `ga`" approach, why do we need these typings at all? I thought that these only provided TypeScript declarations; do they do something special for the closure compiler as well?

Reply:
We'll need this if we want to call `ga` from a `.ts` file, which we don't do currently. In that case, we'll probably need to add a build rule to the tf_imports package that references this file. But I'm ok punting that for now. See also: https://github.com/tensorflow/tensorboard/blob/master/tensorboard/java/org/tensorflow/tensorboard/vulcanize/externs.js#L28

Reply:
SG. Leaving this here for potential future use.